chimney/.jvm/src/test/scala/io/scalaland/chimney/JavaEnumSpec.scala

```diff
@@ -0,0 +1,123 @@
+package io.scalaland.chimney
+
+import utest._
+import io.scalaland.chimney.examples.JavaEnums
+import io.scalaland.chimney.examples.JavaEnums.Colors3
+import io.scalaland.chimney.examples._
+
+object JavaEnumSpec extends TestSuite {
+
+  val tests = Tests {
```

Comment:
Would be nice to see 1-2 test cases that demonstrate using Java enum transformers in some nested context (e.g. as a field of a case class).